tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
-envlist = py27-django{18,19,110,111},py34-django18,py35-django{19,110,111}
+envlist = django{18,19,110,111}
```

Comment:
I noticed that the `pyXX` envs weren't actually being used for anything, so I removed them. I attempted to replicate the same set of python/django combinations in `.travis.yml`